RIOT-OS/RIOT
# boards/nucleo-f722ze: Add periph_can

boards/nucleo-f722ze: Add periph_can support

cpu/stm32: Add CAN support for f722ze board
f722ze board has ONLY 1 CAN interface, fix compiling error which
treats 2 or more CANs for f722ze. Also add loopback mode

sys/include/can: Add loopback operation mode

tests/candev: Add loopback mode for testing purpose

### Contribution description

- stm32f722ze MCU contains CAN 2.0B interface. Makefile.features for board nucleo-f722ze is missing this periph_can.
- Fix a bug in cpu/stm32/include/can_params.h which treats f722ze with more than 1 CAN interfaces.
- For test purpose, add loopback. Due to hardware limitation, I have to use loopback mode to test my changes.


### Testing procedure
First, pin CAN_TX(PD1) and CAN_RX(PD0) shall be connected with 120 ohm. For me, i short the pins for loopback test is good enough
Second, compile candev application to test. in RIOT/ folder, run
`CFLAGS=-DCAN_LOOPBACK_MODE make BOARD=nucleo-f722ze CAN_DRIVER=PERIPH_CAN -C tests/candev flash term`
In pyterm, type `send <data>` to send bytes, and check data is looping back to receive FIFO with command `receive`
My test as following i.e. 

```
2021-03-04 00:45:18,622 # main(): This is RIOT! (Version: 2021.04-devel-931-gbf93d-nucleo-f722ze_add_CAN)
2021-03-04 00:45:18,624 # candev test application
2021-03-04 00:45:18,625 # 
2021-03-04 00:45:18,627 # Initializing CAN periph device
> send 1 2 3 4
2021-03-04 00:45:22,683 # send 1 2 3 4
> 
2021-03-04 00:45:25,078 # send 1 2 3 4
> receive
2021-03-04 00:45:28,133 # receive
2021-03-04 00:45:28,136 # Reading from Rxbuf...
2021-03-04 00:45:28,137 # id: 1 dlc: hx Data: 
2021-03-04 00:45:28,139 # 0x1 0x2 0x3 0x4 
> 
2021-03-04 00:45:29,963 # receive
2021-03-04 00:45:29,965 # Reading from Rxbuf...
```


### Issues/PRs references

Fixes #15711
